lib/Telemetry/src/SensorGps.cpp

```diff
@@ -1,81 +1,119 @@
 #include "SensorGps.h"
 
-char sentenceBuffer[128];
-String completeSentence = "";
+SensorGps::SensorGps(uint8_t updateFrequency) {
+    _updateFrequency = updateFrequency;
+    _gps = new SFE_UBLOX_GNSS();
 
+}
 
-SensorGps::SensorGps(uint16_t updateInterval) {
-    _updateInterval = updateInterval;
+SensorGps::SensorGps(const SensorGps &other){
+    _updateFrequency = other._updateFrequency;
     _gps = new SFE_UBLOX_GNSS();
 }
 
+SensorGps& SensorGps::operator = (const SensorGps &other){
+    if(this == &other) return *this;
+    _updateFrequency = other._updateFrequency;
+    _gps = other._gps;
```

Comment:
I think this line may be problematic if we have a situation like so:

```cpp
SensorGps a = new SensorGps(...);
SensorGps b = new SensorGps(...);
a = b;
delete b;
// a now holds a reference to an internal `_gps` that has been freed
```

It may be more correct to just pass in the `SFE_UBLOX_GNSS` dependency that's constructed externally; there isn't necessarily a problem with passing in physical resources across multiple instances of a sensor.